internal/pkg/agent/cmd/enroll.go

```diff
@@ -285,6 +288,10 @@ func buildEnrollmentFlags(cmd *cobra.Command, url string, token string) []string
 		args = append(args, "--daemon-timeout")
 		args = append(args, daemonTimeout.String())
 	}
+	if enrollTimeout != 0 {
```

Comment:
Should this be `> 0` for consistency?

Reply:
Nope, we want to pass a negative here so the `install` command can disable the timeout as well